packages/agents/src/index.ts

```diff
@@ -1049,12 +1052,12 @@ export class Agent<Env, State = unknown> extends Server<Env> {
     this.broadcast(
       JSON.stringify({
         type: "cf_agent_mcp_servers",
-        mcp: this._getMcpServerStateInternal(),
+        mcp: this.getMcpServerState(),
       })
     );
   }
 
-  private _getMcpServerStateInternal(): MCPServersState {
+  getMcpServerState(): MCPServersState {
```

Comment:
I landed this, but thinking... how do you feel about just calling this "getMcpServers"?

Reply:
I'm cool with that, that's probably a better name

Reply:
ok done https://github.com/cloudflare/agents/pull/279